.gitignore

```diff
@@ -1,3 +1,3 @@
 run/
-bin/
+bin/adam
```

Comment:
You can build for alternate OSes, so let's leave `bin/` as the entire directory that is in `.gitignore`. No reason we cannot put `eve-embedded.sh` in the root directory here.

If you really want it in `bin/`, then let's make the output of the builds in `dist/` and save `eve-embedded.sh` in bin/`, but make sure to change the `Makefile` and the README.doc

Reply:
I am not a big fan of lose files in the root folder, so I added it to scripts. Hope this is OK @deitch

Reply:
Sure, that is just as fine.